pkgs/applications/emulators/bsnes/ares/default.nix

```diff
@@ -80,17 +76,16 @@ stdenv.mkDerivation (finalAttrs: {
     "local=false"
     "openmp=true"
     "prefix=$(out)"
-    "-C desktop-ui"
   ];
 
   env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.isDarwin "-mmacosx-version-min=10.14";
 
-  meta = with lib; {
+  meta = {
     homepage = "https://ares-emu.net";
     description = "Open-source multi-system emulator with a focus on accuracy and preservation";
-    license = licenses.isc;
-    maintainers = with maintainers; [ Madouura AndersonTorres ];
-    platforms = platforms.unix;
+    license = lib.licenses.isc;
+    maintainers = with lib.maintainers; [ Madouura AndersonTorres ];
```

Comment:
There is no difference between those two and using with lib here is not an anti pattern.